docs/timeline/index.html

```diff
@@ -1024,10 +1024,23 @@ <h2 id="Configuration_Options">Configuration Options</h2>
       </td>
     </tr>
 
+    <tr class='toggle collapsible' onclick="toggleTable('optionTable','tooltipOnItemUpdateTime', this);">
+      <td><span parent="tooltipOnItemUpdateTime" class="right-caret"></span> tooltipOnItemUpdateTime</td>
+      <td>Object/Boolean</td>
+      <td><code>false</code></td>
+      <td>Show a tooltip on updating an item's time. Note: <code>editable.updateTime</code> must be <code>true</code></td>
+    </tr>
+    <tr parent="tooltipOnItemUpdateTime" class="hidden">
+      <td class="indent">template</td>
```

Comment:
Well, it's in the `tooltipOnItemUpdateTime` so I don't think we need to change the name.